frontend/src/components/VContentSwitcher/VSearchTypeButton.vue

```diff
@@ -9,7 +9,7 @@
     v-bind="$attrs"
     @click="$emit('click')"
   >
-    <VIcon :name="searchType" class="h-6 w-6" />
+    <VIcon :name="searchType" />
```

Comment:
Just noting for other reviewer(s), this is safe to remove as it is the default value.